client/handle_http.go

```diff
@@ -2381,6 +2421,94 @@ func (te *TransferEngine) walkDirUpload(job *clientTransferJob, transfers []tran
 	return err
 }
 
+// This function performs the ls command by walking through the specified directory and printing the contents of the files
+func listHttp(ctx context.Context, remoteObjectUrl *url.URL, directorUrl string, namespace namespaces.Namespace, token string, flagOptions map[string]bool) (err error) {
+	// Get our directory listing host
+	dirListHost, err := getDirListHost(ctx, remoteObjectUrl, namespace, directorUrl)
+	if err != nil {
+		return
+	}
+	log.Debugln("Dir list host: ", dirListHost.String())
+
+	auth := &bearerAuth{token: token}
+	client := gowebdav.NewAuthClient(dirListHost.String(), auth)
+
+	// XRootD does not like keep alives and kills things, so turn them off.
```

Comment:
I think this might just be an old comment that got copy/pasted over and over. I'll work to remove it because I'm also unsure what it means and must have missed it